### ISSUE TITLE
Instance methods

### ISSUE THREAD

**Issue by twist900** [vadimdemedes/interactor#14]
First, this is a great module, I used interactors in Rails and it was a convenient way of code organization. 
My question is how would you add instance methods to the interactor besides` run()`.  The example bellow will result in an error as in super.run() `this` is set to the `new Interactor()` object for which someHelperMethod() and anotherHelperMethod() are not defined.
```
class SaveUser extends Interactor {
  someHelperMethod() {
  }

  anotherHelperMethod() {
  }

  run() {
   // Will give error `someHelperMethod` is not defined
   let valueA = this.someHelperMethod();
   let valueB = this.anotherHelperMethod();
   return User.save({ valueA, valueB });
  }
}
```

**Comment by vadimdemedes** [comment]
Hey, thanks, glad you are liking it!

You've raised a valid point and the fix for it is to assign context to `this.context`, rather than `this`, which makes class methods unavailable. I hope I will have some time during the weekend to fix this.

**Comment by vadimdemedes** [comment]
Fixed in `master`, will release as a major update. Now you have full access to `this` of your instance. Context is now passed as an argument to `run()` or you can access it via `this.context`.

**Comment by vadimdemedes** [comment]
Just released `2.0.0`!
